news/_posts/2020-03-11-improvements-on-contributors-board.md

```diff
@@ -0,0 +1,75 @@
+
+This is why it is time to improve the Contributors' board. Let's see what is on the schedule already.
+
+
+## The current plan
+
+### First step: quick wins
+
+If you go on [contributors.prestashop.com](http://contributors.prestashop.com/), it is easy to notice a few issues:
+
+- Some contributors like PrestaShop employees are not listed, although they do a huge job on the open source project since the very beginning, and this work is currently hidden
+- It is not possible to see the entire list of contributors, without clicking on "see more", or on page numbers
+- The board lists only contributions to the core; but the PrestaShop software is also made of modules, and of course, there are other very important contributions like specifications, documentation, blog articles, and many more. 
+
+So, it became obvious that the first step must be to show the work of everyone involved in the making of the PrestaShop open source software project.
+
+In the weeks to come, a few improvements will be in production:
+
+1. Remove the current filter to hide some contributors, in order to see everybody. Like the current version, the number of commits will be used, that part doesn't change
+2. Instead of just counting commits on the core repository, it is time to count commits on all the repositories; this is more easy, now that the PrestaShop GitHub's organisation is focused only on the open source project
+3. The page will be a bit modified to be fully scrollable, without hiden parts, with the aim to make it easy to find someone 
+
+This will be the first improvements, and the development work has been started already. The result is subject to a evolve a bit at the beginning, depending on how data will be aggregated, and also because the cleanup of the GitHub organization is not yet complete (e.g: there are still some old partners modules that needs to be archived). However, the new board should quickly give a comprehensive view of the entire contributors team over years.
+
+### Second step: new features and look'n feel
+
+New contents are currently studied:
+
+- A dedicated "profile page" for each contributor, with the list of repositories and other useful information
+- A way to show or split the leader board per component (core, modules, ...)
+- A Company leaderboard (the current one is only for individuals)
+- A dedicated "profile page" for each company, with the same purpose
+
+Also, please note that a few mockups have been done in 2019 to see how to make the look'n feel more modern and add other interesting boards and features. This should be implemented later this year, depending on the availability of a front developer.
+
+
+## A few questions left
+
+There are still a few questions that need some work, as the target is to create a good way to show the work of contributors:
+
+- How to manage contributors that have worked for many companies over time, in order to count company contributions?
```

Comment:
Also separate individual contributions from company contributions in case they contribute during their own time.

Reply:
If they are freelancers, no problem, it's like a company. We "just" have to take in account the period of time, this will already be a challenge.
However, this is always tricky to separate a contribution of an employee during work time or own time, particularly if this is their job to develop on a software. The best practice to date in IP management is to consider the employer anyway, for most of the time, their is something about that in the contract.

Remember the nginx case recently in Russia?